requests/__init__.py

```diff
@@ -86,6 +87,25 @@ def check_compatibility(urllib3_version, chardet_version):
 except ImportError:
     pass
 
+# Check for older version of pyOpenSSL
+def check_pyopenssl(cryptography_version, pyopenssl_version):
+    # Only applies to pyOpenSSL
+    if sys.modules['OpenSSL'].__title__ == 'pyOpenSSL':
+        # Warning only applies for Cryptography < 1.3.4
+        if list(map(int, cryptography_version.split('.'))) < [1, 3, 4]:
+            # Warning only applies for pyOpenSSL < 16.0.0
+            assert list(map(int, cryptography_version.split('.'))) < [16, 0, 0]
```

Comment:
This will always be false since no relesaed version of cryptography is `16.0.0` or higher. I think this was meant to be `pyopenssl_version`

Reply:
Yeah, it's supposed to be `pyopenssl_version`, I can't believe I missed that typo!